src/plugins/mpath.c

```diff
@@ -156,360 +184,295 @@ gboolean bd_mpath_is_tech_avail (BDMpathTech tech, guint64 mode, GError **error)
  * Tech category: %BD_MPATH_TECH_BASE-%BD_MPATH_TECH_MODE_MODIFY
  */
 gboolean bd_mpath_flush_mpaths (GError **error) {
-    const gchar *argv[3] = {"multipath", "-F", NULL};
-    gboolean success = FALSE;
-    gchar *output = NULL;
+    struct dmmp_mpath **mps = NULL;
+    struct dmmp_context *ctx = NULL;
+    uint32_t mp_count = 0;
+    uint32_t i = 0;
```

Comment:
Please use the *GLib* types here.

Reply:
These are for `libdmmp` function only. Why we need `GLib`ize them?